photon-client/src/components/settings/ApriltagControlCard.vue

```diff
@@ -0,0 +1,95 @@
+<script setup lang="ts">
+import CvSlider from "@/components/common/cv-slider.vue";
+import { useCameraSettingsStore } from "@/stores/settings/CameraSettingsStore";
+import { useStateStore } from "@/stores/StateStore";
+import { useSettingsStore } from "@/stores/settings/GeneralSettingsStore";
+import { Euler, Quaternion } from "three";
+
+function quatToEuler(quat) {
+  console.log(quat);
+  quat = new Quaternion(quat.X, quat.Y, quat.Z, quat.W);
+  return new Euler().setFromQuaternion(quat, "ZYX");
+}
+
+// Convert from radians to degrees.
+const degrees = function (radians) {
+  return (radians * 180) / Math.PI;
+};
+</script>
+
+<template>
+  <v-card dark class="mb-3 pr-6 pb-3" style="background-color: #006492">
+    <v-card-title>Apriltag Layout</v-card-title>
+    <div class="ml-5">
+      <p>Field width: {{ (useSettingsStore().currentFieldLayout.field.width * 3.28084).toFixed(2) }} ft</p>
+      <p>Field length: {{ (useSettingsStore().currentFieldLayout.field.length * 3.28084).toFixed(2) }} ft</p>
+
+      <!-- Simple table height must be set here and in the CSS for the fixed-header to work -->
+      <v-simple-table fixed-header height="100%" dense dark>
+        <template #default>
+          <thead style="font-size: 1.25rem">
+            <tr>
+              <th class="text-center">ID</th>
+              <th class="text-center">X, m</th>
+              <th class="text-center">Y, m</th>
+              <th class="text-center">Z, m</th>
+              <th class="text-center">θ<sub>x</sub>, &deg;</th>
+              <th class="text-center">θ<sub>y</sub>, &deg;</th>
+              <th class="text-center">θ<sub>z</sub>, &deg;</th>
```

Comment:
Specify X, Y, and Z angles

Reply:
How do you mean

Reply:
Write X, Y, and Z angles out explicitly like how it is in the targeting tab instead of just putting theta.